Firestore/Example/FuzzTests/FuzzingResources/Serializer/Corpus/ConvertTextToBinary.sh

```diff
@@ -46,12 +49,12 @@ do
   fi
 
   # Run the conversion.
-  echo "Converting file: $file_name (type: $message_type)"
-  echo "$file_content" \
-    | ${SRCROOT}/../../build/external/protobuf/src/protobuf-build/src/protoc \
-    -I${SRCROOT}/../../Firestore/Protos/protos \
-    -I${SRCROOT}/../../build/external/protobuf/src/protobuf/src \
-    --encode=google.firestore.v1beta1."$message_type" \
+  echo "Converting file: ${file_name} (type: ${message_type})"
+  echo "${file_content}" \
+    | "${SRCROOT}/../../build/external/protobuf/src/protobuf-build/src/protoc" \
+    -I"${SRCROOT}/../../Firestore/Protos/protos" \
+    -I"${SRCROOT}/../../build/external/protobuf/src/protobuf/src" \
+    --encode=google.firestore.v1beta1."${message_type}" \
     google/firestore/v1beta1/document.proto \
-    | tee "$binary_protos_dir"/"$file_name" > /dev/null
+    | tee "${binary_protos_dir}"/"${file_name}" > /dev/null
```

Comment:
Nit: I think it should be possible to use just one pair of quotes for the whole string:
```
tee "${binary_protos_dir}/${file_name}"
```

Reply:
Done.